app/assets/locales/android_translatable_strings.txt

```diff
@@ -752,6 +752,9 @@ permission.sms.install.message=CommCare would like to scan your recent messages
 permission.form.gps.title=Permission to capture location during form entry
 permission.form.gps.message=CommCare would like to capture your location and provide it to forms you complete.
 permission.acquire.required=Acquire Required Permissions
+permission.camera.title=Permission for Camera
+permission.camera.message=In order to capture a image, Commcare needs permission to use your device camera.
+permission.camera.denial.message=Permission denied for camera. Please enable via 'Settings -> Apps -> CommCare -> Permissions -> Camera'

```

Comment:
Agree though I was not able to find any M+ device with a different flow. Also we have one another `install.perms.denied.message` string which mentions the same path. Still I am not 100% sure and can remove the specific path from both strings if we prefer.

Reply:
Ah I forgot that this will only ever come up on M+ devices. That makes me more comfortable with leaving it, though I'm still not certain. I just asked on mobile dev chat to see if we can come to a consensus.